cves/kernel/CVE-2014-8173.yml

```diff
@@ -147,10 +149,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: caught and reported by sasha levin
```

Comment:
Where was this developer from? What organization are they from? How did they find it?